forem/forem
# Redesigned home feed auth CTA

## What type of PR is this? (check all applicable)


## Description

ThisPR updates the UI for home-feed auth CTA.

### Before vs After

<img width="400" alt="Screenshot 2022-10-27 at 5 27 45 PM" src="https://user-images.githubusercontent.com/9396084/198279581-73e6602d-8ae6-4256-a42f-2917ee7bc240.png"> <img width="400" alt="Screenshot 2022-11-03 at 6 04 56 PM" src="https://user-images.githubusercontent.com/9396084/199725302-15513b0d-7058-4d9b-9af3-b2ffeaa6d99a.png">

## Related Tickets & Documents



- Related Issue #18646 
- Closes #18646 

## QA Instructions, Screenshots, Recordings

**How to test?**
- Visit http://localhost:3000/ but make you are not signed in
- Scroll the feed and notice that there is `Create Account / Login` CTA item in feed.

**Output across various screen sizes**

<img width="324" alt="Screenshot 2022-11-03 at 6 03 36 PM" src="https://user-images.githubusercontent.com/9396084/199725687-c279aa45-d683-44c9-ac93-0b6d1cce2969.png"> <img width="382" alt="Screenshot 2022-11-03 at 6 03 58 PM" src="https://user-images.githubusercontent.com/9396084/199725694-6fd09545-c4b3-4c40-a536-bca40cb24922.png">

<img width="764" alt="Screenshot 2022-11-03 at 6 04 25 PM" src="https://user-images.githubusercontent.com/9396084/199725696-b75762ee-ae65-40c3-9688-f4f0841e1a9f.png">
<img width="774" alt="Screenshot 2022-11-03 at 6 04 43 PM" src="https://user-images.githubusercontent.com/9396084/199725701-d9a75b37-5426-4c39-a7bf-76fe17369b4c.png">
<img width="871" alt="Screenshot 2022-11-03 at 6 04 56 PM" src="https://user-images.githubusercontent.com/9396084/199725704-66d82bed-0990-4f26-a9f1-393cc6ef8f07.png">



### UI accessibility concerns?

No

## Added/updated tests?

- [x] No, because it contains only UI changes

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

No
